README.md

```diff
@@ -77,6 +78,20 @@ We use [Jest] to test the app. You can run it by executing:
 make test
 ```
 
+### Linting
+
+We lint the app with [ESLint]. You can run it by:
+
+```shell
+make lint
+```
+
+You can fix problems, where possible, by executing:
+
+```shell
+make fix
```

Comment:
not too clear this is about the linting

Reply:
The command name? `fix-lint`? `lint-fix`?

Reply:
Looks like can escape characters in targets, so can use `lint:fix` (which matches `package.json`).

Reply:
Hmm, maybe not.

Reply:
https://github.com/libero/article-page/pull/4/commits/6665c26b0f580d64b9ad25fa3291fef445f5f8de

Reply:
FWIW `make lint:fix` works for me.